oh_staff_ui/models.py

```diff
@@ -287,3 +287,61 @@ class ItemCopyrightUsage(models.Model):
     type = models.ForeignKey(
         CopyrightType, on_delete=models.PROTECT, blank=False, null=False
     )
+
+
+class RelationType(models.Model):
```

Comment:
Since `RelationType` seems to have just one real value, `relatedTo`, this probably could be eliminated if we don't expect more of these types are needed.  But see other comment about `DescriptionType` vs. `RelationType` confusion.

Reply:
I'm going to remove `Relation` and related table altogether, there is no data in existing DLCS database and the relationships on implicit in the parent/child structure of the `ProjectItem` relationships